Tests/Marketplace/zip_packs.py

```diff
@@ -114,61 +114,110 @@ def remove_test_playbooks_from_signatures(path, filenames):
+    Args:
+        zip_path: path containing all the packs copied from the storage bucket
         gcp_path: The path of the packs in the GCP bucket.
-        destination_path: The path to download the packs to.
-        branch_name: The branch name of the build.
         circle_build: The number of the circle ci build.
-
     Returns:
-        zipped_packs: A list of the downloaded packs paths and their corresponding pack names.
+        A list of dictionaries containing each pack name and it's zip path.
+        [{'Slack': 'content/packs/slack.zip'}, {'qualys': 'content/packs/qualys.zip'}]
     """
     zipped_packs = []
-    with ThreadPoolExecutor(max_workers=MAX_THREADS) as executor:
-        for pack in os.scandir(PACKS_FULL_PATH):  # Get all the pack names
-            if pack.name in IGNORED_FILES:
-                continue
 
-            if gcp_path == BUILD_GCP_PATH:
-                pack_prefix = os.path.join(gcp_path, branch_name, circle_build, 'content', 'packs', pack.name)
-            else:
-                pack_prefix = os.path.join(gcp_path, branch_name, circle_build, pack.name)
-
-            if not branch_name or not circle_build:
-                pack_prefix = pack_prefix.replace('/builds/content', '')
-
-            # Search for the pack in the bucket
-            blobs = list(storage_bucket.list_blobs(prefix=pack_prefix))
-            if blobs:
-                blob = get_latest_pack_zip_from_blob(pack.name, blobs)
-                if not blob:
-                    logging.warning(f'Failed to get the zip of the pack {pack.name} from GCP')
-                    continue
-                download_path = os.path.join(destination_path, f"{pack.name}.zip")
-                zipped_packs.append({pack.name: download_path})
-                logging.info(f'Downloading pack from GCP: {pack.name}')
-                executor_submit(executor, download_path, blob)
-                sleep(1)
-                if os.path.exists('/home/runner/work/content-private/content-private/content/artifacts/'):
-                    logging.info(f"Copying pack from {download_path} to /home/runner/work/content-private/"
-                                 f"content-private/content/artifacts/packs/{pack.name}.zip")
-                    shutil.copy(download_path,
-                                f'/home/runner/work/content-private/content-private/content/artifacts/'
-                                f'packs/{pack.name}.zip')
-            else:
-                logging.warning(f'Did not find a pack to download with the prefix: {pack_prefix}')
+    if gcp_path == BUILD_GCP_PATH:
+        zip_path = os.path.join(zip_path, 'packs')  # directory of the packs
+    else:
+        zip_path = os.path.join(zip_path, circle_build)  # directory of the packs
+
+    dir_entries = os.listdir(zip_path)
+    print(f'zip path is: {zip_path}')
+    print(f"List of packs in repo is {', '.join(dir_entries)}")
```

Comment:
either remove prints or use logging module

Reply:
Sure, changed it.